Terminal.Gui/Views/ColorPicker2.cs

```diff
@@ -210,7 +230,7 @@ public class ColorPicker2 : View
     /// </summary>
     public event EventHandler<ColorEventArgs> ColorChanged;
 
-    private bool updating;
+    private bool _updating;
```

Comment:
These kinds of variables always feel hacky to me. Is this stricktly needed, or is there another way of ensuring updates are not problematic?

Reply:
There are 7 views who can each independently trigger a change in color plus the root public property.

- when a bar position changes x3
- when a bar text field changes (currently tied to leave not text changed) x3
- when hex text field changes (again with leave event)

In each case we must update the other 6 (or all 7 if public property is source of change I.e. user calls API direct)

I will have a look, maybe there is a way to handle it with a `SetValueWithoutRaisingEvent` method instead.

Reply:
Implemented in [d6eb6c3](https://github.com/gui-cs/Terminal.Gui/pull/3604/commits/d6eb6c387f25d72f5dbb94992a38d22e2cb0ef5a)